samples/bookinfo/routing-rules/route-rule-details-v2.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: details-route
+spec:
+  hosts:
+  - details
+  http:
+  - route:
+    - destination:
+        name: details
+        subset: v2
+      weight: 100
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: details-destination
+spec:
+  name: details
+  subsets:
+  - name: v2
+    labels:
+      version: v2
```

Comment:
We need both old and new subsets here.. i mean v1 and v2. For this purpose, I suggest we define all subsets in one file..